components/core/tests/tests_login_test.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from UserManager import *
+import unittest
+
+
```

Comment:
Is there a reason why there are two blank lines between code blocks instead of one?

Reply:
No these are left because of keeping the code easy to read but if you think i will remove the extra line.